autoload/composer.vim

```diff
@@ -39,7 +39,8 @@ endfunction
 function! s:json_decode(string) abort
   try
     if exists('*json_decode')
-      return json_decode(a:string)
+      let string = type(a:string) == type([]) ? join(a:string) : a:string
```

Comment:
I'm not sure that it matters for our purposes, but strictly speaking this should be `join(a:string, "\n")`.